.github/workflows/perf_micro.yml

```diff
@@ -76,7 +84,7 @@ jobs:
           fetch-depth: 0
           submodules: recursive
           repository: tarantool/tarantool
-          ref: ${{ inputs.submodule && 'master' || github.ref }}
+          ref: ${{ inputs.submodule && 'master' || env.PERF_COMMIT }}
```

Comment:
Side note: don't quite get this change.
Is it necessary for the target_pull_request, or is this an unrelated fix?

Reply:
Without hunk, workflow will be executed in base branch commit.

![screen-1734426556](https://github.com/user-attachments/assets/adf16382-9d92-45f7-b318-1c661ab550a7)
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target